addon/mixins/popup.js

```diff
@@ -63,7 +63,7 @@ export default Mixin.create({
   didCreateLayer() {
     this._super(...arguments);
     if (this.get('hasBlock')) {
-      this._popup = this.L.popup({}, this._layer);
+      this._popup = this.L.popup(this.get('popupOptions'), this._layer);
       this._popup.setContent(this.get('destinationElement'));
       this._layer.bindPopup(this._popup, this.get('popupOptions'));
```

Comment:
To be honest, I never looked into the details. I think I misunderstood one of the comments in #66 and assumed it had to work with an empty object passed it.

It seems to work locally. Since ember-leaflet is passing in a `L.Popup`, instance to `bindpopup()` it should always work.  

see: https://github.com/Leaflet/Leaflet/blob/master/src/layer/Layer.Popup.js#L25 that it